pypresence/utils.py

```diff
@@ -1,5 +1,5 @@
-"""Util functions that are needed but messy."""
-import asyncio
+"""Util functions that are messy but needed."""
+# import asyncio
```

Comment:
Why comment it? Might as well remove it.

Reply:
Look at your comment at the bottom of the same script